checker/checker.ml

```diff
@@ -211,7 +211,7 @@ let print_usage_coqtop () =
 let usage () =
   print_usage_coqtop ();
   flush stderr;
-  exit 1
+  exit 0
```

Comment:
This function is also used for incorrect arguments, eg `coqchk -Q` (-Q needs 2 arguments after it) so unconditional `exit 0` is incorrect.

Reply:
Did miss that usage is also used for wrong arguments.

Reply:
I changed the function to get the exit code as argument, which returns the old behavior for the non-help case.